src/EnergyPlus/ZoneContaminantPredictorCorrector.cc

```diff
@@ -2141,6 +2141,11 @@ namespace ZoneContaminantPredictorCorrector {
 				for ( NodeNum = 1; NodeNum <= ZoneEquipConfig( ZoneEquipConfigNum ).NumExhaustNodes; ++NodeNum ) {
 					ExhMassFlowRate += Node( ZoneEquipConfig( ZoneEquipConfigNum ).ExhaustNode( NodeNum ) ).MassFlowRate / ZoneMult;
 				} // NodeNum
+				if ( !( ZoneEquipConfig( ZoneEquipConfigNum ).AirLoopNum > 0 || ZoneAirMassFlow.EnforceZoneMassBalance || AirflowNetworkNumOfExhFan > 0 ) ) {
+					ExhMassFlowRate -= ZoneEquipConfig( ZoneEquipConfigNum ).ZoneExh / ZoneMult; // Exclude all exhaust fan flow here
+				} else {
+					ExhMassFlowRate -= ZoneEquipConfig( ZoneEquipConfigNum ).ZoneExhBalanced / ZoneMult; // Balanced exhaust flow assumes there are other flows providing makeup air such as mixing or infiltration, so subtract it here
+				}
```

Comment:
@mjwitte Sorry. I forgot it. I am going to make the same changes in the contaminant corrector.